linux_pipeline/Jenkinsfile_azure_lis_hotfix_performance

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env groovy
+def PerfTestList = [
+	["TCPNetworkPerformance_Multiclient", "PERF-NETWORK-TCP-THROUGHPUT-MULTICLIENTS-NTTTCP-Synthetic,PERF-NETWORK-TCP-THROUGHPUT-MULTICLIENTS-NTTTCP-SRIOV","westus2"],
+	["TCPNetworkPerformance_SingleClient","PERF-NETWORK-TCP-LATENCY-Synthetic,PERF-NETWORK-TCP-LATENCY-SRIOV,PERF-NETWORK-TCP-SINGLE-CONNECTION-THROUGHPUT-SYNTHETIC,PERF-NETWORK-TCP-SINGLE-CONNECTION-THROUGHPUT-SRIOV",
+	"westus2"],
```

Comment:
Please check that new line is not creating any problem.

Reply:
test names and regions are extracted, newline is not creating a problem